docs/metrics.md

```diff
@@ -112,6 +122,17 @@ Most or all of the test metrics overlap with metrics that are sent in telemetry
 | `processWorkingSet` | integer | All physical memory (in bytes) used by the process. |
 | `processWorkingSetPrivate` | integer | Amount of non-shared physical memory (in bytes) used by the process. |
 
+## Latency metrics
+Latency metrics are only pushed to Azure Monotor.
+
+| metric name | format | meaning |
+| - | - | - |
+| `LatencyQueueMessageToSendInMilliseconds` | float | Number of milliseconds between a message is queued to send and when it is actually sent. |
+| `LatencySendMessageToServiceAckInSeconds` | float | Number of seconds between when a message is sent and when the corresponding `serviceAck` is received back from the service. |
+| `LatencyAddReportedPropertyToServiceAckInSeconds` | float | Number of seconds between when a reported property is added and when the corresponding `serviceAck` is received back from the service. |
+| `LatencyRemoveReportedPropertyToServiceAckInSeconds` | float | Number of seconds etween when a reported property is removed and when the corresponding `serviceAck` is received back from the servie. |
+| `LatencyBetweenC2dInSeconds` | float | Number of seconds between consecuitive C2d messages. |
```

Comment:
misspelled:
consecutive

Reply:
I need to get an editor with a built-in spell checker. :)